modules/040-node-manager/hooks/update_approval.go

```diff
@@ -215,55 +215,69 @@ func (ar *updateApprover) approveUpdates(input *go_hook.HookInput) error {
 		}
 
 		countToApprove := concurrency - currentUpdates
+		approvedNodes := make([]updateApprovalNode, 0, countToApprove)
```

Comment:
need using map for deduplication

Reply:
we decided that this cycle does not need and all of this checks will be placed to  approveDisruptions func